class "javax.annotation.ManagedBean"'s signer information does not match signer information of other classes in the same package

```
cd $HOME/.klab/releases/develop/cli
"$HOME/.klab/jre16/bin/java" -Xms512M -Xmx2048M -server --illegal-access=permit -Dfile.encoding=UTF-8 -Dserver-port=8283 -cp cli-0.11.0-SNAPSHOT.jar:lib/* org.integratedmodelling.klab.k.Main -network
```
This fails with
```
java.lang.IllegalStateException: Unable create a default ApplicationContext instance, you may need a custom ApplicationContextFactory
	at org.springframework.boot.ApplicationContextFactory.lambda$static$0(ApplicationContextFactory.java:56)
	at org.springframework.boot.SpringApplication.createApplicationContext(SpringApplication.java:596)
	at org.springframework.boot.SpringApplication.run(SpringApplication.java:335)
	at org.integratedmodelling.klab.engines.modeler.base.Modeler.startNetworkServices(Modeler.java:106)
	at org.integratedmodelling.klab.engines.modeler.base.Modeler.startNetworkServices(Modeler.java:83)
	at org.integratedmodelling.klab.clitool.CliRuntime.startNetwork(CliRuntime.java:67)
	at org.integratedmodelling.klab.clitool.CliRuntime.initialize(CliRuntime.java:56)
	at org.integratedmodelling.klab.clitool.console.TermConsole$1$2.run(TermConsole.java:280)
Caused by: java.lang.SecurityException: class "javax.annotation.ManagedBean"'s signer information does not match signer information of other classes in the same package
	at java.base/java.lang.ClassLoader.checkCerts(Unknown Source)
	at java.base/java.lang.ClassLoader.preDefineClass(Unknown Source)
	at java.base/java.lang.ClassLoader.defineClass(Unknown Source)
	at java.base/java.security.SecureClassLoader.defineClass(Unknown Source)
	at java.base/jdk.internal.loader.BuiltinClassLoader.defineClass(Unknown Source)
	at java.base/jdk.internal.loader.BuiltinClassLoader.findClassOnClassPathOrNull(Unknown Source)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClassOrNull(Unknown Source)
	at java.base/jdk.internal.loader.BuiltinClassLoader.loadClass(Unknown Source)
	at java.base/jdk.internal.loader.ClassLoaders$AppClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.ClassLoader.loadClass(Unknown Source)
	at java.base/java.lang.Class.forName0(Native Method)
	at java.base/java.lang.Class.forName(Unknown Source)
	at org.springframework.util.ClassUtils.forName(ClassUtils.java:284)
```

--- a/products/cli/pom.xml
+++ b/products/cli/pom.xml
@@ -23,6 +23,7 @@
 							<overWriteReleases>false</overWriteReleases>
 							<overWriteSnapshots>false</overWriteSnapshots>
 							<overWriteIfNewer>true</overWriteIfNewer>
+							<excludeArtifactIds>javax.annotation</excludeArtifactIds>
 						</configuration>
 					</execution>
 				</executions>

